installer.sh

```diff
@@ -0,0 +1,10 @@
+sudo apt-get -y install libpng12-dev
+cd ~
+git clone https://github.com/AndrewFromMelbourne/raspi2png.git
+cd raspi2png
+sudo make
+sudo chmod 755 raspi2png
```

Comment:
Any reason to chmod and mv in the script rather than using the following

`sudo make install`

Reply:
No reason. I do not know very much about the make command or building C stuff in general.

Reply:
Hi, I have been busy recently. Sorry that it has taken a while. I was just going to do this myself. I would really prefer the _make_ as two lines.

```
make
sudo make install
```

that way the files in the _pi_ home directory don't belong to _root_.